packages/headless/src/features/facets/range-facets/numeric-facet-set/interfaces/request.ts

```diff
@@ -2,6 +2,9 @@ import {BaseRangeFacetRequest} from '../../generic/interfaces/request';
 import {CurrentValues, Type} from '../../../facet-api/request';
 import {FacetValueState} from '../../../facet-api/value';
 
+/**
+ * The options defining a value to display in a `NumericFacet`.
+ */
 export interface NumericRangeRequest {
```

Comment:
The name of this interface isn't so fitting, but it's already exported in the controller `index.ts`.